.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.6','3.7','3.8']
+        experimental: [false]
+        include:
+          - os: ubuntu-20.04
+            python-version: '3.9-dev'
+            experimental: true
+
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install system packages
+      run: |
+        sudo apt-get install g++
+        sudo apt-get install cmake
+        sudo apt-get install bison
+        sudo apt-get install flex
+        sudo apt-get install python2-minimal
+
+    - name: Install Dependencies
+      run: pip install -r requirements.txt
+
+    - name: Run Tests
+      if: matrix.python-version != '3.9-dev'
+      run: python build_scripts/travis_script.py
+
+    - name: Run Tests for python 3.9
+      if: matrix.python-version == '3.9-dev'
+      run: python build_scripts/travis_script.py || exit 0
```

Comment:
this will make sure the build remains green if it fails while running on python 3.9 only and run normally otherwise for other python versions